### DISCUSSION TITLE
What content should we have in the readme of CryptoEconLab's repository?

### DISCUSSION THREAD

**Discussion by misilva73** [protocol/CryptoEconLab#14] [General]
Opening a discussion for issue #13 about what content should we have in the readme file of this repo. Some guiding principles:

1. Make it easy to keep updated
2. Avoid repetition with other public resources (e.g. websites)

cc @vkalghatgi @dmc314

**Comment by misilva73** [comment]
Some ideas of things to update:
* Add a new section on "How to contribute to the repo", i.e., how to submit code, issues, and discussions.
* Update links to the website (we are still linking the old research one)
* Update mission to align with the website
* Update research focus to align with the website
* Remove the section on team and link to the website instead
* Remove the section on open problems and link to the folder of problems instead (easier to update and maintain)
* "Publications, Talks, & Tutorials" -> what should we link here? hackMD, cryptoecondays, any others?

**Comment by dmc314** [comment]
This is great - thanks for taking initiative! I think our public github readme could shed almost everything that’s on there today.

Much of the content about open problems and projects is outdated, and I think that could be it’s own page. We should get in the habit of doing PRs for any projects that would be useful to us… that way interested interns, postdocs, etc can apply and tailor their applications to stuff that is immediately useful to us.

Any “mission” or “contact us” stuff could be very curtailed, use the up-to-date verbiage from our website, and defer to the website for the main content.

**Reply by misilva73** [reply]
I agree. I will open a PR for these changes shortly

**Comment by dmc314** [comment]
Should we remove the email address at the very bottom? i'm not sure who monitors that inbox. we had talked about a google-form-fill that we could put on the website or here if we need something like that? 
Otherwise we could try to chase down that inbox and see if we can forward it to someone or a group of people...or maybe webhook it into a slack channel or something

**Reply by misilva73** [reply]
Good point. I left it there because it was the only contact we had there before. Happy to drop it

**Reply by misilva73** [reply]
Done: https://github.com/protocol/CryptoEconLab/commit/f733cc4f6361b9ed00bfd5d29e7f1698c3c07f52
